src/en/release/1.10.2.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Release Notes - 1.10.2
+date: 2025-05-27
+linked: true
+meta:
+_edit_last: '4'
+_wpas_done_all: '1'
+---
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+### Highlight
+
+- Support Spark 4.0
```

Comment:
v1.10.2 has the same support as v1.9.4 for Spark 4.0

What works now: Spark SQL engine, KSHC, TPC-H/DS connectors
Not work: Spark extensions, Authz, Lineage